src/test/resources/application.yml

```diff
@@ -9,6 +9,8 @@ spring:
       enabled: false
     initializer:
       enabled: false
+    jdbc:
+      initialize-schema: never
```

Comment:
@siarhei-charniak @alekGbuz when we added this new test PresignedUrlControllerTest - we began to receive strange errors like this:
Access to DialectResolutionInfo cannot be null when 'hibernate.dialect' not set
or this:
java.lang.IllegalStateException: Unable to detect database type
Here is details: https://jenkins-aws.indexdata.com/job/folio-org/job/mod-data-export-worker/job/MODEXPW-330 
Without this new test for controller everething is good, maybe you faced similar situation before?

Reply:
We tried to add initialize-schema: never , but it does not resolve this issue

Reply:
The fix was to change mock of RemoteFileStorage to real object from spring context, so issue resolved, thanks